hibernate-reactive-core/src/main/java/org/hibernate/reactive/provider/service/AbstractReactiveInformationSchemaBasedExtractorImpl.java

```diff
@@ -165,10 +165,16 @@ protected <T> T processCrossReferenceResultSet(
 			String foreignCatalog,
 			String foreignSchema,
 			String foreignTable,
-			ExtractionContext.ResultSetProcessor<T> processor) {
+			ExtractionContext.ResultSetProcessor<T> processor)
+			throws SQLException {
 		// This method has been added as fix for https://hibernate.atlassian.net/browse/HHH-18221
 		// The issue is only for Informix that we don't currently support.
-		throw LOG.notYetImplemented();
+		try ( ResultSet resultSet =
+					  getExtractionContext()
+							  .getJdbcDatabaseMetaData()
+							  .getCrossReference( parentCatalog, parentSchema, parentTable, foreignCatalog, foreignSchema, foreignTable) ) {
+			return processor.process( resultSet );
+		}
```

Comment:
I don't know if this solution is correct. It seems it's using something related to JDBC.
This method is only needed for Informix that we don't support at the moment.

I think we can just close this issue. and let the code as is.
Unless there's a way with the current dialects to call it.